docs/commands/push.md

```diff
@@ -1,93 +1,186 @@
+### Destination
 
-To authenticate to the API registry, you can use the `REDOCLY_AUTHORIZATION` environment variable. It can be set to either your [personal API key](../../workflows/personal-api-keys.md) or to an organization-wide API key (configurable by organization owners in **Redocly Workflows > Org settings > API keys**).
+To properly push your API definition to the Redocly API registry, you need the following information:
 
-Treat the API keys as secrets and work with them accordingly. Consult the documentation for your CI system to learn more about handling secrets:
+- [Organization ID](#organization-id)
+- [API name](#api-name)
+- [API version](#api-version)
 
-- [Travis CI documentation](https://docs.travis-ci.com/user/environment-variables/)
-- [CircleCI documentation](https://circleci.com/docs/2.0/env-vars/)
-- [GitHub Actions documentation](https://docs.github.com/en/free-pro-team@latest/actions/reference/encrypted-secrets)
-- [Jenkins documentation](https://www.jenkins.io/doc/book/pipeline/jenkinsfile/#handling-credentials)
+The format to pass this information is the following: `@organization-id/api-name@api-version`.
 
+#### Organization ID
 
-By default, the `push` command only updates an existing API definition version. If an API with the provided name and version doesn't exist in your organization, it will not be created automatically, and the command will exit with an error exit code.
+To find your organization ID required for the command:
 
-Note that only API definitions with a CI source can be updated with the `push` command. Attempting to update API definitions created from other sources will fail with an error.
+1. Log into Workflows.
+1. Access the **API registry** page.
+1. In your browser's address bar, find the URL of this page.
+1. Inspect the segment after `app.redoc.ly/org/`. This part is your organization ID.
 
-To create a new API and a new API definition version with the `push` command, use the `-u` option:
+For example, if the URL is `app.redoc.ly/org/test_docs`, the organization ID is `test_docs`. When using the `push` command, you would provide this ID as `@test_docs`.
 
+:::warning Note
+The organization ID can differ from the organization name. Owners can change the organization name at any time in the Workflows **Org settings** page, but the organization ID cannot be changed.
+:::
 
-```bash
-openapi push -u test-api-v1.yaml @redocly/test-api@v1 main
-```
+#### API name
+
+To find your API name required for the command:
+
+1. Log into Workflows.
+1. Access the **API registry** page.
+1. Check the list of APIs displayed on this page.
+1. Inspect the title of each list item to the left of the **New version** and **Edit API** action buttons. This title is an API name.
+
+When using the `push` command, you would provide API name after the [Organization ID](#organization-id) separated with the forward slash (`/`). For example, `@test_docs/petstore-api`.
+
+:::info
```

Comment:
This is more than just some sidenote info - it's critical information that can prevent the user from completing the action successfully. I suggest we change it to warning or attention.